Servers/UI/OJS.Servers.Ui/OJS.Servers.Ui.csproj

```diff
@@ -57,6 +57,10 @@
         <None Include="$(SpaRoot)**" Exclude="$(SpaRoot)node_modules\**" />
     </ItemGroup>
 
+    <ItemGroup>
+        <PackageReference Include="MassTransit" Version="8.0.16" />
+    </ItemGroup>
+
```

Comment:
Why do you need MassTransit reference here?

Reply:
Done.